gprofiler/gprofiler_types.py

```diff
@@ -32,6 +33,31 @@ class ProfileData:
 ProcessToProfileData = MutableMapping[int, ProfileData]
 
 
+class ProfilingErrorStack(StackToSampleCount):
+
+    PROFILING_ERROR_STACK_PTRN = re.compile(r".*;\[Profiling .+: .+\]")
+
+    def __init__(self, what: str, reason: str, comm: str):
+        super().__init__()
+        self.update({f"{comm};[Profiling {what}: {reason}]": 1})
```

Comment:
```suggestion
        self.update({f"{comm};[Profiling {what}: {reason}]": 1})
        assert self.is_error_stack(self)
```

Sanity check for your test

Reply:
That's a nice improvement; will help keep the check and format in sync.